NextcloudTalk/NewRoomTableViewController.swift

```diff
@@ -0,0 +1,303 @@
+//
+// Copyright (c) 2024 Ivan Sein <ivan@nextcloud.com>
+//
+// Author Ivan Sein <ivan@nextcloud.com>
+//
+// GNU GPL version 3 or any later version
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+enum NewRoomOption: Int {
+    case createNewRoom = 0
+    case joinOpenRoom
+}
+
+@objcMembers class NewRoomTableViewController: UITableViewController,
+                                               UINavigationControllerDelegate,
+                                               UISearchResultsUpdating {
+
+    var account: TalkAccount
+
+    var indexes: [String] = [""]
```

Comment:
This feels wrong 🤔 Is it to align the section number to the index?

Reply:
This is basically the section we'll use for the "Create a new conversation" and "Join open conversations" options.
We can take another (more self-explainatory) approach sure ;)

Reply:
Not really sure how to solve it easily, without counting differently in all other methods. I'd say we keep it for now like this?!